helm/robusta/values.yaml

```diff
@@ -176,6 +176,14 @@ builtinPlaybooks:
   - image_pull_backoff_reporter: {}
 
 # playbooks for non-prometheus based monitoring that use prometheus for enrichment
+- name: "PodEvicted"
+  triggers:
+  - on_pod_evicted: {}
+  actions:
+  - on_pod_evicted_enricher: {}
```

Comment:
I think it would be best to name it `pod_evicted_enricher` 
the `on_xxxx` is the convention of triggers